CONTRIBUTING.md

```diff
@@ -2,10 +2,19 @@
 Contributing to pydicom
 =======================
 
-This is the summary for contributing code, documentation, testing, and filing
-issues. Please read it carefully to help making the code review process go as
-smoothly as possible and maximize the likelihood of your contribution being
-merged.
+This is the guide for contributing code, documentation and tests, and for
+filing issues. Please read it carefully to help making the code review
+process go as smoothly as possible and maximize the likelihood of your
```

Comment:
"help making" does not sound right.  Could be "help make".

Reply:
Done.

Reply:
It might be just me, but it still reads as "help making"

Reply:
Oops, didn't push the changes...